code/modules/mining/machine_vending.dm

```diff
@@ -48,7 +48,7 @@ var/global/list/minevendor_list = list( //keep in order of price
 	new /datum/data/mining_equipment("KA Range Increase",			/obj/item/borg/upgrade/modkit/range,	10,									750),
 	new /datum/data/mining_equipment("Jaunter",						/obj/item/device/wormhole_jaunter,		20,								750),
 	new /datum/data/mining_equipment("Kinetic Accelerator",			/obj/item/weapon/gun/energy/kinetic_accelerator,	10,					750),
-	new /datum/data/mining_equipment("1000 credits",				/obj/item/weapon/spacecash/c1000,	-1,									800),
+	new /datum/data/mining_equipment("1000 credits",				/obj/item/weapon/spacecash/c100,	-1,									1000),
```

Comment:
help I clicked the 1000 credits option and it spit out 100 credits I've been ripped off